lib/sisimai/data.rb

```diff
@@ -377,8 +377,8 @@ def self.make(data: nil, **argvs)
 
           if o.softbounce.to_s.empty?
             # The value is not set yet
-            textasargv = sprintf('%s %s', p['deliverystatus'], p['diagnosticcode'])
-            textasargv = textasargv.gsub(/\A[ ]/, '')
+            textasargv = p['deliverystatus'] + ' ' + p['diagnosticcode'])
```

Comment:
__[Reek]__

[Syntax] This file has unexpected token tRPAREN


<!-- Comment by [SideCI](https://sideci.com) -->